src/components/MentionSuggestions.tsx

```diff
@@ -86,6 +86,7 @@ function MentionSuggestions({prefix, mentions, highlightedMentionIndex = 0, onSe
                                 type={item.icons[0].type}
                                 fill={isIcon ? theme.success : undefined}
                                 fallbackIcon={item.icons[0].fallbackIcon}
+                                accountID={item.icons[0].id}
```

Comment:
We might want to do optional chaining here?

Reply:
the check in line 80:
`  {item.icons && !!item.icons.length && (`

seems to guarantee icons always being there, so no need to optional chain here IMO